### ISSUE TITLE
license validation fails after WP Engine migration to staging

### ISSUE THREAD

**Issue by worldwildweb** [wp-media/wp-rocket#6136]
**Before submitting an issue please check that you’ve completed the following steps:**
- Made sure you’re on the latest version ✅
- Used the search feature to ensure that the bug hasn’t been reported before ✅

**Describe the bug**
WP Engine changes the `consumer_email` value in the `wp_rocket_settings` option from the `wp_options` table during the migration to staging which requires the user or support to intervene to resolve this issue.

**To Reproduce**

**Expected behavior**
The `consumer_email` value shouldn't be modified so the license is validated without manual intervention.

**Screenshots**
![Screenshot 2023-09-02 at 12 11 00 copy](https://github.com/wp-media/wp-rocket/assets/1954918/c7dfb9c8-67f6-4834-a255-593bbe675b2d)

**Additional context**

**Acceptance Criteria (for WP Media team use only)**
Clear instructions for developers, to be added before the grooming

**Comment by webtrainingwheels** [comment]
Think it might be the same as: https://github.com/wp-media/wp-rocket/issues/2804

**Comment by piotrbak** [comment]
@DahmaniAdame what do you think about hashing those values? It'd prevent from doing unintentional rewrites during search and replace when the website domain matches email domain 🤔

**Comment by DahmaniAdame** [comment]
Yes. That would solve it for similar cases linked to hosting services or even cloning or migration plugins.
